### ISSUE TITLE
Wi-Fi Connection switch issue

### ISSUE THREAD

**Issue by prateekamana** [linuxmint/mint20-beta#153]
When wifi connections are switch, the wi-fi connection that I switch to doesn't appear in the panel wi-fi menu, but the previous connection is visible in the menu. It gets pretty confusing as I can't tell which connection I am connected to.

**Comment by clefebvre** [comment]
That's in Cinnamon? If so can you remove the network applet from the panel? After you do so you'll see the nm-applet applet (NetworkManager GTK applet). See if the same happens there.

**Comment by zuliz95** [comment]
Yes mee too... But i'm in Linuxmint 19.3...

I try reinstalling bcmwl kernel.  

sudo apt install --reinstall bcmwl-kernel-source

And now solved 😀

**Comment by prateekamana** [comment]
@clefebvre Yes, this is in Cinnamon. I did remove the network applet from the panel, in the nm-applet the problem doesn't exist.



Also, after last night's updates (I think there was a kernel update too but I'm not sure) wifi stopped working for me, I solved it by installing Rtl8821ce-dkms. My kernel version currently is "5.4.0-37-generic". Should I open a separate issue ?

**Comment by clefebvre** [comment]
dmesg might help you understand why you lose connection at driver level. Let's focus on the cinnamon issue alone. Can you provide screenshots and explain exactly what's going on?

I'll move this issue to the cinnamo repo now.
